### PR TITLE
Fixed code to CakePHP 4.x

### PR DESCRIPTION
**Pull request by steinkel**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -43,13 +43,13 @@
...
-        "cakephp/cakephp": "^3.5.5"
+    "require":{
+        "cakephp/cakephp": "4.x-dev as 4.0.0",
+        "cakephp/plugin-installer": "*"
     },
     "require-dev": {
-        "cakephp/cakephp-codesniffer": "^3.0",
-        "phpunit/phpunit": "^5.7.14|^6.0"
+        "phpunit/phpunit": "^7",
+        "cakephp/cakephp-codesniffer": "dev-master"
```
Comment (ADmad) [inline composer.json]: The coding standard has been updated for Cake 4. This needs to be `dev-next`.

File: tests/bootstrap.php [hunk 2]
```diff
@@ -29,9 +30,13 @@
...
+}
 Configure::write('App', [
+    'namespace' => 'CsvView\Test\App',
     'paths' => [
-        'templates' => [dirname(__FILE__) . DS . 'test_app' . DS . 'TestApp' . DS . 'Template' . DS],
+        'templates' => [dirname(__FILE__) . DS . 'test_app' . DS . 'templates' . DS],
     ]
-]);
-Plugin::load('CsvView', ['path' => dirname(dirname(__FILE__)) . DS]);
+ ]);
```
Comment (ADmad) [inline tests/bootstrap.php]: Indentation seems off here.

File: composer.json (hunk 1 above)
Comment (skie) [reply composer.json]: switched to dev-next

File: tests/bootstrap.php (hunk 2 above)
Comment (skie) [reply tests/bootstrap.php]: maybe it was. if you see i removed code, not added

**Pull request merged**